NitroxClient/Communication/DeferringPacketReceiver.cs

```diff
@@ -109,14 +112,26 @@ public void CellLoaded(VisibleCell visibleCell)
                 Queue<Packet> deferredPackets;
                 if (deferredPacketsByAbsoluteCell.TryGetValue(visibleCell.AbsoluteCellEntity, out deferredPackets))
                 {
+                    Log.Debug("Loaded {0}; found {1} deferred packet(s):{2}\nAdding it back with high priority.",
+                        visibleCell,
+                        deferredPackets.Count,
+                        deferredPackets.PrefixWith("\n\t"));
                     while (deferredPackets.Count > 0)
                     {
-                        Log.Debug("Found deferred packet... adding it back with high priority.");
                         Packet packet = deferredPackets.Dequeue();
                         receivedPackets.Enqueue(EXPIDITED_PACKET_PRIORITY, packet);
                     }
                 }
             }
         }
+
+        public void Flush()
```

Comment:
Isn't `Flush` supposed to send all outstanding packets to the server, and deal with incoming packets immediately? How about some different terminology (such as `Clear`), or make this function do what people assume `Flush` means?

Reply:
I'd started with `Purge` but do not recall why I ended up back here. Maybe because I intended to flush at the point it is called? I am only clearing it out because we don't have reconnects ironed at all. I am indifferent about the name at this point personally.

Reply:
I've seen the discussion on Discord regarding removing outstanding packets in the queue and deferred structure. We shall be adding reconnect functionality soon enough, so this function can stay the way it is for now.